homeassistant/components/snapcast/media_player.py

```diff
@@ -67,7 +68,10 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
     platform.async_register_entity_service(SERVICE_UNJOIN, {}, handle_async_unjoin)
     platform.async_register_entity_service(
         SERVICE_SET_LATENCY,
-        {vol.Required(ATTR_LATENCY): cv.positive_int},
+        {
+            vol.Required(ATTR_LATENCY): cv.positive_int,
+            vol.Required(ATTR_ENTITY_ID): cv.entity_id,
```

Comment:
This is already done by the platform entity service helper:
https://github.com/home-assistant/core/blob/ed7ac3c1f78ab17094c4228f1da1302097a56dbe/homeassistant/helpers/entity_platform.py#L507-L508

Reply:
So entity_id is required by default? This code is unnecessary?

Reply:
Yes, either entity_id or area_id is required.

Reply:
As said, the entity service register already takes care of this.
entity_id is optional, as an area_id can be supplied alternatively. However, from an entity perspective, it doesn't matter, as the service method is called by Home Assistant if the service call matches.

This PR is actually not fixing anything, as nothing is broken. 👍

Reply:
Cool! 👍